Meme/Meme/MemeTemplatesViewController.swift

```diff
@@ -24,9 +24,8 @@ class MemeTemplatesViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         self.viewModel.requestTemplates { [weak self] templates in
-            templates.forEach({ template in
-                self?.getMeme(template)
-            })
+            let _ = templates.compactMap({return self?.getMeme($0)})
+            return nil
```

Comment:
Is this what you had in mind? Not sure complier is requiring me to return a nil in this case. Any advice @vhart ?